src/ng-generate/table-prompter/index.ts

```diff
@@ -524,6 +523,14 @@ function getUserConfigQuestions(allAnswers: any, tree: Tree, options: Schema): Q
         default: false,
     };
 
+    const requestOptionalMaterialTheme = {
+        type: 'confirm',
+        name: 'getOptionalMaterialTheme',
+        message: 'Do you want to add indigo.pink material theme?',
```

Comment:
Do you want to add the Angular Material theme? (Indigo Pink Theme)

Reply:
done